server/web/sponsor/index.hbs

```diff
@@ -0,0 +1,37 @@
+<h1>Sponsor</h1>
+
+<p>
+  Development of jsPerf is generously supported by contributions from individuals and corporations. If you are benefiting from jsPerf and would like to help keep the project financially sustainable, please donate to a core contributor.
+</p>
```

Comment:
Perhaps we could link to the open issues list, so that people can pick their favorite issue and sponsor the person who’s assigned to work on it, rather than randomly picking one of the core contributors in the list below?